_config.yml

```diff
@@ -204,7 +204,7 @@ toc:
 #creative_commons:
 
 sidebar:
-  # Sidebar Position, available value: left | right (only for Pisces | Gemini).
+  # Sidebar Position, available value: left | right .
```

Comment:
Only for Pisces | Gemini | Mist. unavailable for Muse.

Reply:
sorry, i forgot it, and i just fix Scheme.Mist.

Reply:
Muse scheme tested?

Reply:
wait, tonight update.

Reply:
@zhuzhuyule take in attention: in Mist scheme temporairly removed `float: right` style to compatability with sub-menu feature. See here: https://github.com/theme-next/hexo-theme-next/pull/156/files#diff-90c3d2a8522cb0e6c0ac99af79e65607R9